client/containers/DashboardOverview/ContentOverview.js

```diff
@@ -14,19 +18,35 @@ const propTypes = {
 
 const ContentOverview = (props) => {
 	const { overviewData } = props;
-	const {
-		scopeData: {
-			elements: { activeTargetType },
-		},
-	} = usePageContext();
+	const { scopeData } = usePageContext();
+	const useCollectionStateObject = useCollectionState(scopeData);
```

Comment:
We should avoid naming the return value of these hooks with values starting with `use`:

- At a glance, the variables look like they themselves are hooks
- I don't think it's important to know that these values come from a hook. In modern React, the sources of truth for most stateful values _do_ come from a hook, and we should feel free to swap hooks in and out without the variable names bound to these values having to change as well!

Instead I would suggest:
- Destructuring at the hook's callsite, e.g. `const {xyz, updateXyz} = useXyz()`, and passing the object state and the updater functions as separate props to child components that consume them. This feels like 'idiomatic React' in the sense that data flows in as one prop, and the functions that update the data are a second prop — it also signals intent about whether a child component ought to be able to update state or merely consume it. This can be pattern-matched from the APIs of the built-in hooks, i.e. we get `[state, updateState]` from `useState()` and `[state, dispatch]` from `useReducer()`, and we're expected to destructure the return value. Most libraries that expose hooks do something similar. 
- (Less preferred) giving the returned state values a name like `xyzState` and retrieving the values as `xyzState.xyz` and `xyzState.updateXyz`.

Reply:
Agreed. Fixed with refactor in latest commit.